lib/commonAPI/mediacapture/ext/platform/android/src/com/rho/camera/CameraObject.java

```diff
@@ -206,20 +214,18 @@ public void stopPreview() {
     public ISize setPreviewSize(int width, int height) {
     	Camera camera = getCamera();
         Camera.Parameters parameters = camera.getParameters();       
-        Intent intent = new Intent();
         List<android.hardware.Camera.Size> sizes = camera.getParameters().getSupportedPictureSizes();        
         android.hardware.Camera.Size maxSize=sizes.get(0);       
         if(getActualPropertyMap().containsKey("desiredWidth") || getActualPropertyMap().containsKey("desiredHeight")){
         	int desired_width = Integer.parseInt(getActualPropertyMap().get("desiredWidth"));
-            int desired_height = Integer.parseInt(getActualPropertyMap().get("desiredHeight"));            
+            int desired_height = Integer.parseInt(getActualPropertyMap().get("desiredHeight"));               
         	if(desired_width > 0 && desired_width <= maxSize.width && desired_height > 0 && desired_height <= maxSize.height){        	
         	 Camera.Size previewSize = getOptimalPreviewSize(parameters.getSupportedPictureSizes(), desired_width, desired_height);
         	 Logger.T(TAG, "Selected size: " + previewSize.width + "x" + previewSize.height);             
              parameters.setPreviewSize(previewSize.width, previewSize.height);
         	}
-        	else{
-        		//Handle error message
-        		intent.putExtra("error", "Please enter a valid desired value");        		
+        	else{ 
+        		parameters.setPreviewSize(320, 240);
```

Comment:
Why you decided to not inform developer about wrong parameter value? The behavior at hi-res devices will be strange?...